packages/deprecation-crawler/src/lib/constants.ts

```diff
@@ -15,3 +15,5 @@ export const enum CRAWLER_MODES {
   'SANDBOX' = 'SANDBOX_MODE',
   'CI' = 'CI',
 }
+export const MD_GROUP_OPENER = '<!-- ruid-groups';
```

Comment:
Should [these](https://github.com/timdeschryver/deprecation-manager/blob/master/packages/deprecation-crawler/src/lib/output-formatters/markdown/generate-group-based-formatter.ts#L9-L10) be removed?

Reply:
done